src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -377,6 +407,8 @@ private BibtexEntry cleanup(BibtexEntry entry) {
                 fullName = fullName.replaceAll(" on", " ").replace("  ", " ");
             }
 
+            Pattern publicationPattern = Pattern.compile("(.*), \\d*\\.*\\s?(.*)");
```

Comment:
Ok makes sense, reverted it.  But does that mean we should do the same for all the regex's here?  There are a lot of them...